src/store/modules/export/selectors.ts

```diff
@@ -0,0 +1,68 @@
+import { createSelector } from 'reselect';
+import { ExportEntry, StoreExportEntry } from './types';
+import { State } from '../../types';
+
+import * as moment from 'moment';
+
+const getExportEntries = (state: State) => state.export.rows;
+const getCurrentExportEntryUUID = (state: State) => state.activeRows.activeExportEntryUUID;
+
+const decimals = 1;
+
+const maybeGetActiveExportEntry = createSelector(
+  getCurrentExportEntryUUID,
+  getExportEntries,
+  (uuid, getExportEntries) => getExportEntries.find(e => e.uuid === uuid),
+);
+
+/**
+ * Selector for getting the active export entry
+ */
+export const getActiveExportEntry = createSelector(
+  maybeGetActiveExportEntry,
+  (maybeExportEntry) => {
+    // TODO: Re-evaluate this
+    if (!maybeExportEntry) {
+      const empty: StoreExportEntry = {
+        type: 'export',
+        datetime: '',
+        transportId: '',
```

Comment:
What's `transportId`?

Reply:
in the database we generalized it in case in future implementations its not a truck.